tools/packaging/static-build/cache_components.sh

```diff
@@ -74,6 +74,14 @@ cache_virtiofsd_artifacts() {
 	create_cache_asset "${virtiofsd_tarball_name}" "${current_virtiofsd_version}" "${current_virtiofsd_image}"
 }
 
+cache_shim_v2_artifacts() {
+	local shim_v2_tarball_name="kata-static-cc-shim-v2.tar.xz"
+	local shim_v2_file="${script_dir}/../../../VERSION"
+	local current_shim_v2_version="$(cat ${shim_v2_file})"
```

Comment:
The current_shim_v2_version could be something like:
current_shim_v2_version="$(get_last_modification "${repo_root_dir}/src/runtime")-${get_from_kata_deps lagangues.go.meta.newest...)-$(get_from_kata_deps languages.rust...)".

In this way we can ensure that if the code changes, or the tools used to build the code changes, we force a rebuild.